### PR TITLE
Added ERROR as allowed loglevel

### LINKED ISSUE #1597: Add flag to run golem with different log severity
We can change log severity by editing logging.ini, but it will be more convenient if we are able to just change this with just adding specific flag that will change the severity. It's suggested that it will have three levels: 

* normal (no flag): log warnings and higher
* verbose (--verbose, -v): all logs (debug and higher)
* quiet (--quiet, -q): only critical and exceptions

### PR DESCRIPTION
**Pull request by maaktweluit**
Resolves #1597

### CODE REVIEW

File: golem/core/common.py [hunk 1]
```diff
@@ -14,6 +14,13 @@
 MAX_CPU_WINDOWS = 32
 MAX_CPU_MACOS = 16
 
+ALLOW_LOGLEVELS = [
```
Comment (jiivan) [inline golem/core/common.py]: `ALLOWED_LOGLEVELS`?

Comment (maaktweluit) [reply golem/core/common.py]: Good one, fixed :)

**Review by jiivan — APPROVED** [review]

**Comment by jiivan** [comment]
@maaktweluit please remember to change help text for `--loglevel` also.

**Comment by maaktweluit** [comment]
@jiivan Good catch! updated it :)

**Pull request merged**